examples/multi_window.py

```diff
@@ -7,6 +7,8 @@
 """
 
 def load_html():
+    import time
+    time.sleep(1)
```

Comment:
The actual fault is in my implementation of `get_key_instance`. It returns
`None` when no window has focus. I did it on purpose, thinking that the
way things are, no one can get back into the Python interpreter without
focusing and interacting with any pywebview window. This one is an
exceptional case, so instead of returning `None`, may be we should
raise an `Exception`?

This is not a problem when the app is bundled into executable, I have
verified it on Mac. It's only when running from the command-line that
this pops up. I think adding this delay is the best way to get away with
it. After all it's just a demo. 😉